anthem/lyrics/modules.py

```diff
@@ -8,35 +8,37 @@
 def uninstall(ctx, module_list):
     """ uninstall module """
     if not module_list:
-        raise AnthemError(u"You have to provide a list of "
-                          "module's name to uninstall")
+        raise AnthemError(
+            u"You have to provide a list of module's name to uninstall"
+        )
 
-    mods = ctx.env['ir.module.module'].search([('name', 'in', module_list)])
+    mods = ctx.env["ir.module.module"].search([
+        ("name", "in", module_list), ("state", "!=", "uninstalled")
```

Comment:
FYI Skipping the uninstalled modules. Useful when running with `MARABUNTA_FORCE_VERSION`

CC: @JevinD